release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -156,6 +156,7 @@ With this release, there are several updates to the *Administrator* perspective
 
 * The {product-title} web console displays a `ConsoleNotification` if the cluster is upgrading. Once the upgrade is done, the notification is removed.
 * A *_restart rollout_* option for the `Deployment` resource and a *_retry rollouts_* option for the `DeploymentConfig` resource are available on the *Action* and *Kebab* menus.
+* You can view a list of supported clusters on the *All Clusters* dropdown of the *Topology* view. The supported products include {product-title}, {product-title} Service on AWS (ROSA), Azure Red Hat OpenShift (ARO), ROKS, and {product-dedicated}. 
```

Comment:
```
You can view a list of supported clusters on the All Clusters dropdown of the Topology view
```

Do we have better wording for this sentence?  Maybe we can just say following?
```
You can view a list of supported clusters on the `All Clusters` dropdown list
```

Reply:
Yeah that would work! I was just thinking of pointing them in some sort of direction where they could get the list, but I think it is pretty straightforward.